src/Pages/Product/Product.js

```diff
@@ -9,35 +12,84 @@ export class Product extends Component {
     this.state = {
       productDetailImages: [],
       productInformation: {},
+      productReviewData: [],
+      quantitydefaultValue: 1,
+      moreView: false,
     };
   }
 
   componentDidMount() {
-    fetch('/data/wearImagesData.json')
+    fetch('/data/electronicImagesData.json')
       .then(res => res.json())
       .then(productPageData => {
         this.setState({
           productDetailImages: productPageData,
         });
       });
+
+    // fetch('http://192.168.0.4:8000/products/440')
+    //   .then(res => res.json())
+    //   .then(productInfoData => {
+    //     this.setState({
+    //       productInformation: productInfoData.data.product,
+    //     });
+    //   });
+
     fetch('/data/productInfoData.json')
       .then(res => res.json())
       .then(productInfoData => {
         this.setState({
-          productInformation: productInfoData,
+          productInformation: productInfoData.data.product,
+        });
+      });
+
+    fetch('/data/reviewData.json')
+      .then(res => res.json())
+      .then(reviewData => {
+        this.setState({
+          productReviewData: reviewData,
         });
       });
   }
 
+  plusQuantity = () => {
+    this.setState({
+      quantitydefaultValue: Number(this.state.quantitydefaultValue) + 1,
+    });
+  };
+
+  minusQuantity = () => {
+    if (this.state.quantitydefaultValue > 1) {
+      this.setState({
+        quantitydefaultValue: this.state.quantitydefaultValue - 1,
+      });
+    }
+  };
+
+  handleQuantityInput = e => {
+    this.setState({
+      quantitydefaultValue: e.target.value,
+    });
+  };
+
+  handleMoreView = () => {
```

Comment:
```suggestion
  toggleMoreView = () => {
```
토글 기능이니까 토글이라고 적어주는게 좋아보입니다

Reply:
넵 :)